src/ui/controls/nav-button/style.less

```diff
@@ -1,10 +1,18 @@
 @import "../../theme.less";
 
 .nav-button {
-    background-image: @icon-right;
-    background-position: center right 0.25rem;
+    &.external-link {
+        background-image: @icon-external;
+        background-size: 1.5rem;
+        background-position: center right;
+    }
+    &:not(.external-link) {
```

Comment:
I think you can leave the default options for .nav-button, because `.external-link` will override them anyway